pipelines/utils.py

```diff
@@ -59,7 +59,7 @@ def parse_model_batch_dir_name(model_batch_dir_name: str) -> dict:
 
 def get_all_forecast_dirs(
```

Comment:
I think it would be better to revert this change and do the list conversion when calling the function.